src/test/groovy/com/intuit/graphql/orchestrator/batch/DownstreamQueryModifierSpec.groovy

```diff
@@ -237,17 +246,22 @@ class DownstreamQueryModifierSpec extends Specification {
                 .build()
 
         when:
+        AstTransformer astTransformer = new AstTransformer()
         Field newA1 = (Field) astTransformer.transform(a1, subjectUnderTest)
 
         then:
         // test '{ a1 { af1 reqdField} }', expected
         @SuppressWarnings("rawtypes")
         List<Selection> actualSelections = newA1.getSelectionSet().getSelections()
+
         actualSelections.size() == 2
-        toFieldNameSet(actualSelections) == ["af1", "reqdField"]
+        toFieldNameSet(actualSelections).sort() == ["af1", "reqdField"]
     }
 
-    void canRenameQueryFields() {
+    def "can Rename Query Fields"() {
+        given:
+        AstTransformer astTransformer = new AstTransformer()
```

Comment:
should we put in class level?  to avoid repeating

Reply:
Updated accordingly.